libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -86,7 +93,57 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     });
   }
 
+  private getContentMaxHeight(): number {
+    const ionModalWrapper = this.elementRef.nativeElement.parentElement;
+    const ionModal = ionModalWrapper.parentElement;
+    const ionModalPaddingTop = this.windowRef
+      .getComputedStyle(ionModal)
+      .getPropertyValue('padding-top');
+
+    let availableSpace =
+      document.documentElement.clientHeight -
+      parseInt(ionModalPaddingTop) -
+      (this.ionContentElement.nativeElement.getBoundingClientRect().top -
+        this.elementRef.nativeElement.getBoundingClientRect().top);
```

Comment:
Consider extracting to variables for readability:
```suggestion
      (contentWrapperTop - modalWrapperTop);
```